client/components/platform-checkout/save-user/container.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const Container = ( { children, isBlocksCheckout } ) => {
```

Comment:
Mayisha shared her findings on 1447-gh-Automattic/woopay#issuecomment-1339026618 regarding the approach we should take when developing this.

I went a bit further and found out that it's not just the `FormStep` that's not exported, but we can't import anything from `@woocommerce/base-components/cart-checkout` as it would not be found.

>We can copy that existing component and make out own wrapper to render our remember me section as a stepped block but doing so would create UI mismatch if that component gets changed in any future version of the blocks plugin.

I went down on this approach for two reasons:

- We already rely on Blocks checkout classes for this feature [here](https://github.com/Automattic/woocommerce-payments/blob/develop/client/checkout/platform-checkout/index.js#L25-L32).
- The other possibility I see is integrating it via the Blocks plugin itself, like it's mentioned there. However, IMO, I don't see as a viable option for a small feature like this. This feature has changed a lot, and I don't think the required effort for implementing such approach worths this long shot.